analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -197,11 +197,13 @@ abstract class PackageManager {
                 } catch (e: Exception) {
                     e.showStackTrace()
 
+                    val relativePath = analyzerRoot.relativize(definitionFile.toPath())
```

Comment:
How about making `analyzerRoot` a `File` (similar to like we had it before afeac51dd83ac22c78d71a4a8c684c6457b0a82b) and use Kotlin's [File.relativeTo()](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/java.io.-file/relative-to.html)?

Reply:
Done.